.travis.yml

```diff
@@ -2,15 +2,15 @@ language: python
 sudo: false
 services:
   - mysql
+  - postgresql
 python:
-- '3.4'
-- '3.6'
 - '3.7'
+- '3.8'
 env:
-- DJANGO_VERSION=1.8
-- DJANGO_VERSION=1.9
-- DJANGO_VERSION=1.10
-- DJANGO_VERSION=1.11
+- DJANGO_VERSION=1.11 DATABASE_URL=postgres://postgres@127.0.0.1/dbq
+- DJANGO_VERSION=2.2 DATABASE_URL=postgres://postgres@127.0.0.1/dbq
```

Comment:
There's a nicer way to do variables which are the same in all environments:

https://docs.travis-ci.com/user/environment-variables/#global-variables